_includes/docker_platform_matrix.md

```diff
@@ -1,16 +1,20 @@
+| [Microsoft Windows Server 2016](/engine/installation/windows/docker-ee.md) |                                                               |                                                               |                                                               | [{{ green-check }}](/engine/installation/windows/docker-ee.md) |                                                               |                                                              |
+| [Oracle Linux]({{ install-prefix }}/docker-ee/oracle.md)                   |                                                               |                                                               |                                                               | [{{ green-check }}]({{ install-prefix }}/docker-ee/oracle.md)  |                                                               |                                                              |
+| [Red Hat Enterprise Linux]({{ install-prefix }}/docker-ee/rhel.md)         |                                                               |                                                               |                                                               | [{{ green-check }}]({{ install-prefix }}/docker-ee/rhel.md)    | [{{ green-check }}]({{ install-prefix }}/docker-ee/rhel.md)   | [{{ green-check }}]({{ install-prefix }}/docker-ee/rhel.md)  |
+| [SUSE Linux Enterprise Server]({{ install-prefix }}/docker-ee/suse.md)     |                                                               |                                                               |                                                               | [{{ green-check }}]({{ install-prefix }}/docker-ee/suse.md)    | [{{ green-check }}]({{ install-prefix }}/docker-ee/suse.md)   | [{{ green-check }}]({{ install-prefix }}/docker-ee/suse.md)  |
+| [Ubuntu]({{ install-prefix }}/ubuntu.md)                                   | [{{ green-check }}]({{ install-prefix }}/docker-ce/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ce/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ce/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ee/ubuntu.md)  | [{{ green-check }}]({{ install-prefix }}/docker-ee/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ee/ubuntu.md)|
 
-> **Note**: Support for IBM Z on Docker EE is limited to Docker EE Basic, or to
-> worker nodes in Docker EE Standard and Docker EE Advanced. IBM Z nodes cannot
-> function as UCP managers on Docker EE.
\ No newline at end of file
+> Limitations for Docker EE on IBM Z and POWER architectures
+>
+> - Support for IBM Z on Docker EE is limited to Docker EE Basic, or to
+>   worker nodes in Docker EE Standard and Docker EE Advanced. IBM Z nodes cannot
+>   function as UCP managers on Docker EE.
```

Comment:
I'm confused. The above is now true for P, no longer for Z, right?

Reply:
Olivier is correct

Reply:
So Z can run managers now?

Reply:
yes